README.md

```diff
@@ -1,191 +1,107 @@
-# AI Code Reviewer
+# GPT TECH LEADER GIT REVIEWER
+
+[English Version](README_EN.md)
 
 AI 驱动的代码审查助手，自动为 GitHub Pull Requests 提供代码审查和建议。
 
```

Comment:
[From AI Code Reviewer][bot] The addition of a note about submitting MRs for experiencing functionality is helpful, but consider explaining what MR stands for (Merge Request), as this might not be immediately clear to all users.